app/controllers/snapshots_controller.rb

```diff
@@ -18,9 +16,13 @@ def create
   def show
     @snapshot = Snapshot.find(params[:id])
   end
-  
+
   def destroy
     Snapshot.find(params[:id]).destroy
     redirect_to root_path
   end
+
+  def index
+    render text: Snapshot.order(:id).pluck(:id).join(',')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.